js/reducers/approval.js

```diff
@@ -62,20 +61,20 @@ module.exports = function approval(state, action) {
 		case actions.FAIL_APPROVAL_SUBMIT:
 			return state;
 
-		case actions.START_APPROVAL_CANCEL:
+		case actions.SUCCEED_APPROVAL_CANCEL:
 			return _.assign({}, state, {
 				request_sent: false,
 				requested_time: '',
 				approved: false,
```

Comment:
yeah I think so, we can offload that to `deployment.state` I think, as long as that's the place which is always going to be updated from the server after calling cancel, approve, reject etc.